vuepress/docs/next/tutorials/ecr/publish-simple-bundle.md

```diff
@@ -0,0 +1,130 @@
+
+## Create the project folder 
+We'll create a parent folder and then a child bundle folder. In a project generated by the Entando Component Generator the microservice and micro frontend source files also live under the parent folder.
+
+``` sh
+mkdir -p example-bundle/bundle; cd example-bundle/bundle 
+```
+## Add a simple widget
+
+Create a widget directory and descriptor file:
+``` sh
+mkdir widgets
+nano widgets/example-widget.yaml
+```
+
+Now let’s populate the `example-widget.yaml` metadata with some content:
+``` yaml
+code: example-widget
+titles:
+   en: Example Widget
+   it: Widget d'esempio
+group: free
+customUi: <h2>Hi from Example Widget</h2>
+```
+
+## Create the bundle descriptor
+
+The `descriptor.yaml` is the main file processed by the Entando Component Repository and describes all of the components in the bundle. 
+``` sh
+nano descriptor.yaml
+```
+Add the following content to the file. You can use your preferred editor but make sure the YAML indentation matches what you see here.
+``` yaml
+code: example-bundle
+description: This is an example of an Entando bundle
+components:
+  widgets:
+    - widgets/example-widget.yaml
+```
+The component descriptor file name and location is arbitrary since the bundle descriptor explicitly points to it. The usual convention is to group components by type.
+
+## Publish the bundle
+
+You can publish a bundle using the CLI or you can perform the steps by hand.
+### CLI steps
+1. From the project directory, initialize npm and accept all of the defaults. Note: this step is unnecessary for a full Project since the Entando Blueprint handles it. 
+``` sh
+npm init
+```
+2. Now initialize the Entando project and again accept the defaults.
+``` sh
+ent prj init
+``` 
+3. Next initialize the publication system. You'll need the git repository URL and your credentials here. 
+``` sh
+ent prj pbs-init
+```
+4. Now finish publishing the bundle to git. By convention your first version will be `v0.0.1` but this is up to you. 
+``` sh
+ent prj pbs-publish
+``` 
+On subsequent iterations you can run just this command again to quickly push a new version of your Bundle to git.
+
+5. You can now create the custom resource for your bundle and apply it to Kubernetes in one step. You should modify the target namespace parameter (`-n`) if you changed it from the default.
+   ``` sh
+   ent prj generate-cr | ent kubectl apply -n entando -f -
+   ```
+ This convenience method uses the git repository URL and project name from earlier steps to create the custom resource.
+ 
+6. Jump to the section below to finish installing your bundle: [Install the bundle into an application](#install-the-bundle-into-an-application)
+
+
+### Manual steps
+1. From the bundle directory, run the following commands to initialize git and commit the files.
```

Comment:
@w-caffiero-entando , 
1) I'm removing the npm init so that will clean up some of that. In the previous pass I couldn't run ent prj init without it, as you know.
2) I'd rather keep nano for the reasons I mentioned above. I'm a little on the fence on whether including the editor is good but it sure makes the tutorial seamless.
3) Will update the manual steps with a cd. Good catch.

I think this covers your comments but let me know if I missed something.